nwb_conversion_tools/gui/classes/forms_general.py

```diff
@@ -276,8 +276,11 @@ def write_fields(self, data={}):
             str_datetime = data['session_start_time'].strftime('%d/%m/%Y, %H:%M')
             self.form_session_start_time1.setText(str_datetime.split(',')[0])
             self.form_session_start_time2.setText(str_datetime.split(',')[1].strip())
-        if 'experimenter' in data:
-            self.form_experimenter.setText(data['experimenter'])
+        if 'experimenter' in data and data['experimenter'] is not None:
+            if isinstance(data['experimenter'],list):
+                self.form_experimenter.setText(','.join(str(x) for x in data['experimenter']))
+            else:
+                self.form_experimenter.setText(data['experimenter'])
```

Comment:
we need to decide whether we want experimenter data to be an array, string or both. Currently it accepts both.
In the [schema ](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/nwb_conversion_tools/metafile.schema.json#L23)its an array but the code expected it as a string initially.